docs/topics/db/queries.txt

```diff
@@ -970,6 +970,14 @@ field to ``Author``::
     entry.save()
     entry.authors = old_authors # saves new many2many relations
 
+.. note::
```

Comment:
I'd reserve note/warning for items that have less connection to the surrounding text/examples. Since this is directly related to the previous example, I just include it as a regular paragraph, but feel free to do whatever you like.